install/kubernetes/helm/istio/requirements.yaml

```diff
@@ -13,11 +13,11 @@ dependencies:
     repository: file://../subcharts/ingress
   - name: gateways
     version: 1.1.0
-    condition: gateways.enabled
+    condition: gateways.istio-ingressgateway.enabled, gateways.istio-egressgateway.enabled
```

Comment:
what happens to the custom gateways?

Reply:
I 'll need to add it here.  I didn't as you are changing names :)

Reply:
I am not changing gateway names.. My point is, if I have some custom gateways that I want to launch, do I have to edit the requirements file to add this custom gateway and then run helm? Thats a complicated workflow isn't it?

The idea behind gateways is that everyone can spin up their own gateways for variety of purposes. It should not be restricted to just the istio created ones..

Reply:
agreed, that is quite inconvenient, I'll undo this portion.